test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs

```diff
@@ -109,6 +109,7 @@ IEnumerable<Registry> GetAndValidateRegistries()
             this.TestTimeout = TimeSpan.FromMinutes(context.GetValue("testTimeoutMinutes", 5));
             this.Verbose = context.GetValue<bool>("verbose");
             this.ParentHostname = Option.Maybe(Get("parentHostname"));
+            // FQDNS for the Edge Device
```

Comment:
>// FQDNS for the Edge Device [](start = 12, length = 28)

formatting is a little weird here, maybe put it behind the line you're trying to comment?